src/space/delete-test-space-and-environments.ts

```diff
@@ -0,0 +1,23 @@
+import { ClientAPI } from 'contentful-management';
+
+export async function deleteTestSpaceAndEnvironments(
```

Comment:
should we call it `cleanup` - that would also match how it's called in most of the test runners.